packages/frontend/app/(dashboard)/course/[cid]/(settings)/settings/lms_integrations/page.tsx

```diff
@@ -57,6 +61,13 @@ export default function CourseLMSIntegrationPage(props: {
   const [modalOpen, setModalOpen] = useState<boolean>(false)
   const [delModalOpen, setDelModalOpen] = useState<boolean>(false)
   const [isTesting, setIsTesting] = useState<boolean>(false)
+  const [selectedResources, setSelectedResources] = useState<unknown[]>([])
```

Comment:
```suggestion
  const [selectedResources, setSelectedResources] = useState<string[]>([])
```

not sure if this works